src/bcsc-auth/pom.xml

```diff
@@ -86,8 +86,20 @@
 		</dependency>
 		<dependency>
 			<groupId>io.jsonwebtoken</groupId>
-			<artifactId>jjwt</artifactId>
```

Comment:
Upgrade JWT Authentication to its Java 17 version (Jjwt to  jjwt-impl)

Reply:
You can make the same change here as ecrc